packages/core/src/modules/connections/ConnectionsModule.ts

```diff
@@ -86,6 +96,7 @@ export class ConnectionsModule {
       autoAcceptConnection?: boolean
       alias?: string
       mediatorId?: string
+      protocol?: string
```

Comment:
How will this work when protocol is didexchange?

I only accepts an 0160 invitation, but allows a string value protocol.